app/main.py

```diff
@@ -1,27 +1,27 @@
 import logging
 import logging.config
 import os
+from contextlib import asynccontextmanager
 
 import json_logging
 import uvicorn
+from apscheduler.schedulers.background import BackgroundScheduler
+from db_client import run_migrations
 from fastapi import APIRouter, Depends, FastAPI
 from fastapi.middleware.cors import CORSMiddleware
 from fastapi_health import health
 from fastapi_pagination import add_pagination
-from apscheduler.schedulers.background import BackgroundScheduler
 from starlette.requests import Request
-from contextlib import asynccontextmanager
-
-from db_client import run_migrations
```

Comment:
Yay Isort!! Are you interested in adding this to pre commits? Probably separately to this change though as I expect its going to have a lot to say for the codebase as a whole

Reply:
It's rolled in as part of the Trunk config! I have a separate ticket for adding trunk to this repo and fixing autofixable errors, but as I have trunk (& ruff, black etc) setup locally that's why this has snuck in. More on this next week in lunch and learn! 
https://linear.app/climate-policy-radar/issue/PDCT-986/add-trunk-to-nav-backend